test/transforms.jl

```diff
@@ -57,7 +57,7 @@
     vnew, cache = TB.apply(trans, vset)
     @test all(sides(boundingbox(vnew)) .≤ T(1))
     vini = TB.revert(trans, vnew, cache)
-    @test vini == vset
+    @test all(isapprox.(vini, vset))
```

Comment:
```suggestion
    @test all(≈, vini, vset)
```

Reply:
The `all` function does not have this method, just:
```julia
all(itr)
all(f, itr)